crates/store/src/db/sql.rs

```diff
@@ -144,6 +144,43 @@ pub fn select_account(conn: &mut Connection, account_id: AccountId) -> Result<Ac
     account_info_from_row(row)
 }
 
+/// Select account deltas by account id and block range from the DB using the given [Connection].
+///
+/// # Note:
+///
+/// Both `block_start` and `block_end` are included in the range.
+///
+/// # Returns
+///
+/// The account deltas, or an error.
+pub fn select_account_deltas(
+    conn: &mut Connection,
+    account_id: AccountId,
+    block_start: BlockNumber,
+    block_end: BlockNumber,
+) -> Result<Vec<AccountDelta>> {
+    let mut stmt = conn.prepare(
+        "
+        SELECT
+            delta
+        FROM
+            account_deltas
+        WHERE
+            account_id = ?1 AND block_num BETWEEN ?2 AND ?3
+        ORDER BY
+            nonce ASC
+    ",
```

Comment:
Is `BETWEEN` the right operator here? I was thinking it would be more like:
```sql
SELECT
    delta
FROM
    account_deltas
WHERE
    account_id = ?1 AND block_num > ?2 AND block_num <= ?3
ORDER BY
    nonce ASC
```
If we have both sides be inclusive, won't we get some duplication?

Reply:
For integers it's not a big problem. We can provide `from_block + 1` as the range beginning. But I will accept your proposal due to better readability, thanks!